README.md

```diff
@@ -87,6 +87,21 @@ print(list(sorted(ng.optimizers.registry.keys())))
 
 The [optimization documentation](docs/optimization.md) contains more information on how to use several workers, take full control of the optimization through the `ask` and `tell` interface and some pieces of advice on how to choose the proper optimizer for your problem.
 
+## Multiobjective minimization with Nevergrad
+
+Let us minimize x1 and x2 (two objective functions) assuming that values above 2.5 are of no interest.
+```python
+import nevergrad as ng
+
+f = ng.functions.multiobjective_minimization([lambda x: x[0], lambda x: x[1]], bad_values=[2.5, 2.5])
```

Comment:
if `f` is properly instrumented you should do `ng.optimizers.CMA(instrumentation=f.instrumentation, budget=100)`

Reply:
Ok, done!